source/includes/options-mongodrdl-commands.yaml

```diff
@@ -0,0 +1,265 @@
+program: mongodrdl
+name: sample
+directive: commandoption
+description: |
+  Samples the namespaces specified by the
+  :option:`--db <mongodrdl --db>`, optional
+  :option:`--collection <mongodrdl --collection>`, and other
+  :ref:`command line options <mongodrdl-command-line-options>` to
+  output a :ref:`.drdl <drdl>` file. The ``sample`` command
+  encapsulates all of :binary:`~bin.mongodrdl` behavior prior to version
+  2.11.
+
+  .. code-block:: sh
+
+     mongodrdl sample --db <db-name> --collection <collection-name> --out <filename>.drdl
+---
+program: mongodrdl
+name: list-schema-ids
+directive: commandoption
+description: |
+  Lists the :ref:`ObjectId <document-bson-type-object-id>` of each
```

Comment:
This is a bit confusing insofar as "5d72ad695c23a9e3e26e3c85" really a string, as opposed to 'ObjectId("5d72ad695c23a9e3e26e3c85")' which is an ObjectId. It might be simpler to refer to these as "id strings" or just "ids". Bottom line, it's not crucial to the readability or clarity of the page, so it's your call.

Reply:
Yes, that looks good.